pipeline-yaml_5b5744d40428631d7a8940a9.md

```diff
@@ -0,0 +1,548 @@
+  * [About Pipeline YAML](#pipeline-yaml-reference)
+  * [The preamble](#the-preamble)
+    * [version](#version)
+    * [name](#name)
+    * [agent](#agent)
+    * [Preamble example](#preamble-example)
+  * [Blocks](#blocks)
+  * [Task](#task)
+  * [Example of task](#example-of-task)
+  * [Jobs](#jobs)
+  * [Example of jobs](#example-of-jobs)
+  * [The secrets property](#the-secrets-property)
+  * [Example of secrets](#example-of-secrets)
+  * [Complete examples](#complete-configuration-examples)
+  * [The order of execution](#the-order-of-execution)
+  * [See also](#see-also)
```

Comment:
I forgot comment on TOC. Please create TOC where we are listing keywords nested to parent. Like in this example:

```
  * [The preamble](#the-preamble)
     * [version](#version)
     * [name](#name)
     * [agent](#agent)
     * [Preamble example](#preamble-example)
```

I didn't list keywords that live under `agent` but I probably should have.